<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NfoMainWindow</class>
 <widget class="QMainWindow" name="NfoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NFO Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon_enna_128.png</normaloff>icons/icon_enna_128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listOfFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabInfos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>531</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="fileInformation">
     <attribute name="title">
      <string>File information</string>
     </attribute>
     <widget class="QTextEdit" name="fileInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mediaInformatiion">
     <attribute name="title">
      <string>Media information</string>
     </attribute>
     <widget class="QTextEdit" name="nfoInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fanArt">
     <attribute name="title">
      <string>Fanart/Posters</string>
     </attribute>
     <widget class="QGroupBox" name="fanArtPosters">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Fanart/Posters</string>
      </property>
      <widget class="QListWidget" name="fanArtList">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>221</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="fanartDisplayed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>211</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Fanart</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="thumb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>491</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Thumbnails</string>
      </property>
      <widget class="QListWidget" name="thumbList">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>221</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="thumbDisplayed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>211</width>
         <height>171</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Thumbnail</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="launchScan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Files</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="scanProgress">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelListOfFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>List of files </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMovie"/>
    <addaction name="actionTV_Show"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionMovie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Movie</string>
   </property>
  </action>
  <action name="actionTV_Show">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TV Show</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>NfoMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
